src/tests/ftest/aggregation/space_rb.py

```diff
@@ -31,35 +31,34 @@ def verify_space_rb_property(self, pool, expected_space_rb):
                    f"Expected = {expected_space_rb}; Actual = {space_rb}")
             self.fail(msg)
 
-    def run_ior_verify_error(self, pool, container, job_num, block_size, transfer_size, errors):
+    def run_ior_verify_error(self, namespace, pool, container, job_num, errors):
         """Run IOR and verify the error message contains the expected message.
 
         Args:
+            namespace (str): Namespace that defines block_size and transfer_size.
             pool (TestPool): Pool to use with IOR.
             container (TestContainer): Container to use with IOR.
             job_num (int): Indicator to be used in the message.
             block_size (str): Block size parameter for the IOR.
             transfer_size (str): Transfer size parameter for the IOR.
             errors (list): List to collect the errors occurred during the test.
         """
-        ior_cmd = IorCommand()
+        ior_cmd = IorCommand(namespace=namespace)
         ior_cmd.get_params(self)
         ior_cmd.set_daos_params(self.server_group, pool, container.identifier)
         testfile = os.path.join(os.sep, f"test_file_{job_num}")
         ior_cmd.test_file.update(testfile)
-        # Use small transfer size so that data are written to SCM. It's better to hard-code the
-        # values rather than dynamically calculating from the pool size because there's a
-        # requirement about block size must be the multiple of transfer size.
-        ior_cmd.block_size.update(value=block_size)
-        ior_cmd.transfer_size.update(value=transfer_size)
         manager = get_job_manager(test=self, job=ior_cmd, subprocess=self.subprocess)
         manager.assign_hosts(
             self.hostlist_clients, self.workdir, self.hostfile_clients_slots)
+        ppn = self.params.get("ppn", '/run/ior/*')
```

Comment:
This could come from the namespace too
```suggestion
        ppn = self.params.get("ppn", namespace)
```

Reply:
Done